app/functions.R

```diff
@@ -195,13 +198,22 @@ renderExposureHtml <- function(risk, symptoms) {
 # function to create susceptibility HTML output --------------------------------------
 renderSusceptibilityHtml <- function(risk) {
   tags$p(HTML(paste0(
-    "Among people who are the same age, sex, and health status as you and get sick from COVID-19, the risk of hospitalization is ", 
-    formatPercent(risk$hosp_risk),
-    ", the risk of requiring an ICU is ",
-    formatPercent(risk$icu_risk),
-    ", and the risk of not surviving is ",
-    formatPercent(risk$death_risk), "."
+    "Among people who are the same age, sex, and health status as you and get sick from COVID-19, we estimate that one person in a group of ", 
```

Comment:
Could we keep both the probabilities and the frequency-based statement? This will help us check our math so that the error like above is easier to be spotted.

Reply:
Sure, what do you think about this wording? 
![image](https://user-images.githubusercontent.com/63205823/147276610-7b2849a8-afa0-408a-99eb-78d8661b34fc.png)

Reply:
Excellent. Thanks!

Reply:
@mollybair Sorry I didn't realize you were just giving me a preview of the results. It looks good, please push the code.